content/guides/core-concepts/cypress-app.md

```diff
@@ -92,6 +92,32 @@ the performance of your long-running tests.
 
 Again, this analysis comes from the Dashboard.
 
+#### Flake annotation
+
+<Alert type="info">
+
+<strong class="alert-header">What is a flaky test?</strong>
+
+A test is considered to be [flaky](/guides/dashboard/flaky-test-management) when
+it can pass and fail across multiple retry attempts without any code changes.
+
+</Alert>
+
+Specs with flaky tests are flagged with an indicator beside the spec name. This
+allows you to focus your efforts on unreliable and problematic areas of your
+test suite.
```

Comment:
"This means you can easily discern which areas of your application might result in an unreliable user experience." - is this better phrasing? per @mattvCypress

Reply:
@pstakoun I like your proposal.

Reply:
lol, I copied this from your comment on the doc verbatim @mattvCypress